qa/src/test/java/io/camunda/migrator/qa/history/HistoryMigrationSkippingTest.java

```diff
@@ -14,6 +14,7 @@
 import io.camunda.migrator.impl.clients.DbClient;
 import io.camunda.migrator.impl.persistence.IdKeyMapper;
 import io.camunda.search.entities.ProcessInstanceEntity;
+import io.camunda.search.query.DecisionRequirementsQuery;
```

Comment:
It seems most of the changes in these file was due to incorrect formatting (extra indentation being removed by auto format). The new test to review is `shouldSkipDecisionDefinitionWhenDecisionRequirementsIsSkipped`

Reply:
❓ Could you format the whole class as there are 4 spaces instead of 2 in other tests cases too.